Merge/mainnet-readiness.md

```diff
@@ -142,6 +139,7 @@ Most research related to the merge has been completed. This section lists topics
 * [x] Discovery [is there actually anything to do here?]
 * [ ] Execution-layer proof of custody
```

Comment:
Is this still something we'll realistically look at? Seems like we could perhaps check as "wont' do"

Reply:
in research phase. It will definitely not be in the merge